sycl/include/sycl/ext/intel/experimental/esimd/simd_view.hpp

```diff
@@ -70,6 +72,9 @@ class simd_view : public detail::simd_view_impl<BaseTy, RegionTy> {
   simd_view(const simd_view &Other) = default;
   simd_view(simd_view &&Other) = default;
 
+  // Construct a complete view of a vector
+  simd_view(BaseTy &Base) : BaseClass(Base) {}
```

Comment:
Please convert this to a documentation comment, and mention that this does not work for VL=1

Reply:
Thanks. Is VL != 1 limitation still in effect? If so the best way would be to use SFINAE to disable this constructor in this case.

Reply:
I added constructor to scalar specialization. So, VL=1 case works. See this commit: 33eede79f2ca2d92d21dca4b85ca2227bebeb47d